debian/tm-manifesting.install

```diff
@@ -5,7 +5,7 @@ configs/            usr/lib/python3/dist-packages/tmms/
 static/             usr/lib/python3/dist-packages/tmms/
 templates/          usr/lib/python3/dist-packages/tmms/
 tm_cmd/             usr/lib/python3/dist-packages/tmms/
```

Comment:
Do you want to flip these two lines to preserve alphabetical order?

Reply:
Not really if to be honest :) Cause this whole file is not alphabetically sorted anyways.
If you insist then sure. Otherwise - maybe later? (my English doesn't let me sort it quickly).